eclipse/src/saros/ui/wizards/AddProjectToSessionWizard.java

```diff
@@ -567,24 +567,14 @@ public void run(IProgressMonitor monitor)
       final saros.filesystem.IProject adaptedProject =
           ResourceAdapterFactory.create(entry.getValue());
 
-      /*
-       * do not refresh already partially shared projects as this may
-       * trigger resource change events
-       */
       if (!session.isShared(adaptedProject)) project.refreshLocal(IResource.DEPTH_INFINITE, null);
```

Comment:
I don't think this check whether the project is shared is needed anymore, but I am not completely sure, so I left it in for now.